src/include/processor/operator/persistent/index_builder.h

```diff
@@ -67,13 +67,14 @@ class IndexBuilderSharedState {
     friend class IndexBuilder;
 
 public:
-    explicit IndexBuilderSharedState(std::unique_ptr<storage::PrimaryKeyIndexBuilder> pkIndex);
-    void consume() { globalQueues.consume(); }
-    void flush() { globalQueues.flushToDisk(); }
+    explicit IndexBuilderSharedState(storage::PrimaryKeyIndexBuilder* pkIndex)
+        : globalQueues{pkIndex} {}
+    inline void consume() { globalQueues.consume(); }
```

Comment:
Every method defined inside a class definition is implicitly inline. 
![Screenshot_20240103-130453_1](https://github.com/kuzudb/kuzu/assets/41458184/be95c8f5-9b04-4580-89df-15878fe7303a)